src/components/app/list-new/AppList.tsx

```diff
@@ -574,234 +347,149 @@ export default function AppList({ isSuperAdmin, appListCount, isArgoInstalled }:
                                     />
-                                    <span className="filter-divider" />
+                                    <div className="dc__border-right h-16" />
                                 </>
                             )}
-                            <Filter
-                                list={masterFilters.projects}
-                                labelKey="label"
-                                buttonText="Projects"
-                                placeholder="Search Project"
-                                isDisabled={dataStateType === AppListViewType.LOADING}
-                                searchable
-                                multi
-                                type={AppListConstants.FilterType.PROJECT}
-                                applyFilter={applyFilter}
-                                onShowHideFilterContent={onShowHideFilterContent}
-                                dataTestId="projects-filter"
+                            <FilterSelectPicker
+                                placeholder="Project"
+                                inputId="app-list-project-select"
+                                options={projectOptions}
+                                appliedFilterOptions={selectedProjects}
+                                handleApplyFilter={handleUpdateFilters(AppListUrlFilters.project)}
+                                isDisabled={appListFilterLoading || projectListLoading}
+                                isLoading={appListFilterLoading || projectListLoading}
                             />
+                            <div className="dc__border-right h-16" />
                             {serverMode == SERVER_MODE.FULL && (
                                 <>
-                                    <span className="filter-divider" />
-                                    <Filter
-                                        list={masterFilters.environments}
-                                        isDisabled={dataStateType === AppListViewType.LOADING}
-                                        labelKey="label"
-                                        buttonText="Environment"
-                                        searchable
-                                        multi
-                                        placeholder="Search Environment"
-                                        type={AppListConstants.FilterType.ENVIRONMENT}
-                                        applyFilter={applyFilter}
-                                        onShowHideFilterContent={onShowHideFilterContent}
-                                        dataTestId="environment-filter"
-                                    />
+                                    <Tooltip
+                                        content="Remove cluster filters to use environment filter"
+                                        alwaysShowTippyOnHover={!!clusterIdsCsv}
+                                        maxWidth="200px"
```

Comment:
Ig maxWidth is handled inside Tooltip

Reply:
not already handled, added imp class in tooltip